deployment/terraform/variables.tf

```diff
@@ -40,11 +40,19 @@ variable "bastion_ami" {}
 
 variable "bastion_instance_type" {}
 
-variable "fargate_cpu" {
+variable "fargate_app_cpu" {
```

Comment:
Not too thrilled with the name of this variable. Namely because of the variable `app_count`. 

Welcoming any other suggestions.

Reply:
My suggestions would be:

- `fargate_app_cpu`
- `fargate_app_memory`
- `fargate_app_count`

I'd also possibly consider replacing `fargate` with the more generic `ecs`.